### PR TITLE
Add SimpleITK libraries and headers

### PR DESCRIPTION
**Pull request by blowekamp**
This package compiles the SimpleITK libraries which instantiates much
of ITK. The libraries will be reused when building different versions
of Python ( Lua, R ).

### CODE REVIEW

File: recipes/libsimpleitk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+requirements:
+  build:
+    - cmake   >=3.10
+    - ninja
+    - lua     5.1.*
+    - libitk  4.13.*
+    - python  # [win]
+    - vc 9  # [win and py27]
+    - vc 14  # [win and py35>=35]
```
Comment (hmaarrfk) [inline recipes/libsimpleitk/meta.yaml]: I don't think this vc stuff is needed anymore, that is what compiler('c') and 'cxx' do.

File: recipes/libsimpleitk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  run:
+    - libitk 4.13.*
+    - vc 9  # [win and py27]
```
Comment (hmaarrfk) [inline recipes/libsimpleitk/meta.yaml]: When you add compiler c and cxx, they will pin versions of vc for you.

File: recipes/libsimpleitk/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+source:
+  url: https://github.com/SimpleITK/SimpleITK/releases/download/v{{version}}/SimpleITK-{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+    number: 0
+    # Lets first get Linux and OSX working...
+    skip: True # [win and py27]
+    features:
+    - vc9  # [win and py27]
```
Comment (hmaarrfk) [inline recipes/libsimpleitk/meta.yaml]: I'm unsure about the need for these features anymore as well.

**Comment by hmaarrfk** [comment]
You have to split things into `head` and `build`. see https://github.com/conda-forge/opencv-feedstock/blob/master/recipe/meta.yaml for example

**Comment by blowekamp** [comment]
You mean `host` and build?

**Comment by hmaarrfk** [comment]
Seems like you got the host section now

**Comment by blowekamp** [comment]
@conda-forge-manager I think this package is ready to go, please review.

**Comment by scopatz** [comment]
Please ping @conda-forge/staged-recipes  :wink:

File: recipes/libsimpleitk/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+about:
+  home: http::/www.simpleitk.org
+  license: Apache 2.0
+  license_file: LICENSE
+  summary: Development headers and libraries for SimpleITK a simplified interface to the Insight Toolkit.
+
+extra:
+    recipe-maintainers:
+      - blowekamp
+      - zivy
```
Comment (scopatz) [inline recipes/libsimpleitk/meta.yaml]: @zivy Are you OK with being a maintainer?

Comment (zivy) [reply recipes/libsimpleitk/meta.yaml]: I'm ok with being a maintainer.

**Pull request merged**